[DOC] Add deterministic and probabilistic atlases to glossary (#3152)
nilearn/nilearn: doc/glossary.rst

        `Decoding`_ consists in predicting, from brain images, the conditions
        associated to trial.

    Deterministic atlas
        A deterministic atlas is a hard parcellation of the brain into

>         non-overlaping regions, which were obtained by a wide range of methods.

Reviewer: ```suggestion
        non-overlaping regions, that might have been obtained by segmentation or clustering methods.
```
